boards/arm/arduino_nano_33_ble/arduino_nano_33_ble.dts

```diff
@@ -95,6 +95,28 @@
 	status = "okay";
 	sda-pin = <14>; //P0.14
 	scl-pin = <15>; //P0.15
+
+	hts221: hts221@5f {
+		compatible = "st,hts221";
+		status = "disabled";
+		reg = <0x5f>;
+		label = "HTS221";
+	};
+
+	lps22hb: lps22hb-press@5c {
+		compatible = "st,lps22hb-press";
+		status = "disabled";
+		reg = <0x5c>;
+		label = "LPS22HB";
+	};
+
+	apds9960: apds9960@39 {
+		compatible = "avago,apds9960";
+		status = "disabled";
```

Comment:
should these be `status = "okay"`? They won't get enabled unless `CONFIG_SENSOR=y` and `CONFIG_XXX=y` (XXX=sensor driver)

Reply:
Sounds resonable, thank you 👍